src/Npgsql/PostgresException.cs

```diff
@@ -271,6 +271,38 @@ public override bool IsTransient
         /// </summary>
         public NpgsqlStatement Statement { get; internal set; }
 
+        /// <inheritdoc />
+        public override string ToString()
+        {
+            var builder = new StringBuilder(base.ToString());
+            builder.AppendLine().Append("Exception data:");
+
+            void AppendPropertyLine(string propertyName, object propertyValue)
+            {
+                if (propertyValue == null) return;
+                builder.AppendLine().Append(propertyName).Append(": ").Append(propertyValue);
```

Comment:
One thought - it may be a good idea to prepend these lines with tabs, since several log parsing products I know tend to identify each line as a separate message, whereas lines starting with whitespace are considered a continuation of the previous message.

Reply:
How about I indent with the same number of spaces that Exception.ToString() already uses for whitespace ? mixing tabs and spaces feels ugly.